### PR TITLE
Create a notebooks directory

### PR DESCRIPTION
**Pull request by ocefpaf**
I also updated the glider metrics to port all @kbailey-noaa inputs, replaced the old IOOS_BTN notebook with a streamline version that uses the library, and moved the GTS notebook to the notebooks folder.

We need to figure out what to do with the multiple `.py` in the root directory and mint a fresh release with the updated code to so folks can install it and run the notebooks.

### CODE REVIEW

File: IOOS_BTN.ipynb
Comment (MathewBiddle) [inline IOOS_BTN.ipynb:1]: can we keep this for now?

File: ioos_btn_metrics.csv
Comment (MathewBiddle) [inline ioos_btn_metrics.csv:1]: Same here

**Review by MathewBiddle — CHANGES_REQUESTED** [review]

File: IOOS_BTN.ipynb
Comment (ocefpaf) [reply IOOS_BTN.ipynb:1]: Sure. Should we keep it in the new notebooks directory?

File: ioos_btn_metrics.csv
Comment (ocefpaf) [reply ioos_btn_metrics.csv:1]: Sure.

File: IOOS_BTN.ipynb
Comment (MathewBiddle) [reply IOOS_BTN.ipynb:1]: I see it there now. multitasking isn't working.

Comment (ocefpaf) [reply IOOS_BTN.ipynb:1]: I pushed b/c it was there already when I restored but we can move it back to the root directory if you prefer that way.

Comment (MathewBiddle) [reply IOOS_BTN.ipynb:1]: nope, all good here.

**Pull request merged**